ocrd-pagetopdf

```diff
@@ -40,6 +40,19 @@ main () {
         warning "Without a second input file group for images, the original imageFilename will be used"
     fi
     out_file_grp=${ocrd__argv[output_file_grp]}
+    
+    # Remove pre-existing files
+    if [ -d ${out_file_grp} ]; then
+        case ${params['overwrite']} in
```

Comment:
`--overwrite` is now part of the [CLI processor spec](https://ocr-d.de/en/spec/cli#--overwrite). bashlib support is not there yet, unfortunately https://github.com/OCR-D/core/issues/476